src/xcode/ENA/ENA/Source/Views/Common/HtmlTextView.swift

```diff
@@ -47,7 +47,9 @@ class HtmlTextView: UITextView {
 			NSAttributedString.Key.foregroundColor: UIColor.enaColor(for: .textTint)
 		]
 
+		isUserInteractionEnabled = true
 		isEditable = false
+		dataDetectorTypes = [.link, .phoneNumber]
```

Comment:
This PR does not take into account all occurrences of this component. So please double check if this generic application of the data detector types is applicable.

Reply:
moved away from this class, is specific to the AppInformationDetailViewController, which should have the clickable links 😄 the generic component is not touched.